ci/taos/webhook.php

```diff
@@ -71,7 +71,7 @@ function json_config(){
     // Note that you have to use 'print_r' instead of echo function to display decoded json data.
     // print_r ($json_config);
     printf ("[DEBUG] json config: your webhook ID: %s\n<br>", $json_config->github->id);
-    printf ("[DEBUG] json config: your webhook secret: %s\n<br>", substr_replace("$json_config->github->secret","******",3);
```

Comment:
Please do not change a " symbol. it is to handle a string that includes  space (e.g., "my password 38282d").

Reply:
fixed

Reply:
Please, keep one commit  without two commits.